api/src/game_state_server.py

```diff
@@ -27,8 +24,8 @@ class RoomData:
     def __init__(self, room_id: str, initial_connection=None):
         self.room_id = room_id
         self.game_state = {}
-        self._id_to_positions = {}
-        self._positions_to_ids = {}
+        self.id_to_positions = {}
```

Comment:
why aren't these private anymore?

Reply:
Because GameStateServer uses those. We could use getters and setters instead, but...why?